fedora-messaging.spec

```diff
@@ -0,0 +1,132 @@
+%{!?__python2: %global __python2 %__python}
+%{!?python2_sitelib: %global python2_sitelib %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")}
+
+%global pkgname fedora-messaging
+%global srcname fedora_messaging
+%global desc \
+Tools and APIs to make working with AMQP in Fedora easier.
+
+
+Name:           %{pkgname}
+Version:        1.0.0
+Release:        0.1.a1%{?dist}
+Summary:        Set of tools for using Fedora's messaging infrastructure
+
+License:        GPLv2+
+URL:            https://github.com/fedora-infra/fedora-messaging
+Source0:        https://github.com/fedora-infra/%{name}/archive/v%{version}a1/%{srcname}-%{version}a1.tar.gz
```

Comment:
You can also just do %{url}/archive... here if you want. It doesn't matter to me either way.

Reply:
nifty! :-)